### DISCUSSION TITLE
.NET6 is published

### DISCUSSION THREAD

**Discussion by SekiBetu** [kohsuke/winsw#889] [Show and tell]
Just a reminder

**Comment by viceice** [comment]
dotnet 5 is EOL since May 10, 2022

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

So even the v2 version should get an update to net6 with is a lts release again.

I can provide a PR for v2 if it will be accepted

**Comment by nxtn** [comment]
2.12.0 is now on .NET 6.
